x/evm/types/msg.go

```diff
@@ -219,6 +220,11 @@ func (msg *MsgEthereumTx) GetMsgs() []sdk.Msg {
 	return []sdk.Msg{msg}
 }
 
+// TODO(boodyvo): should be implemented if we use
+func (msg *MsgEthereumTx) GetMsgsV2() ([]protov2.Message, error) {
```

Comment:
Need to check in more details. It was added in the PR with other changes:
```
(x/authx) https://github.com/cosmos/cosmos-sdk/pull/15284 NewKeeper now requires codec.Codec.
- types/tx.Tx no longer implements sdk.Tx.
- sdk.Tx now requires a new method GetMsgsV2().
- sdk.Msg.GetSigners was deprecated and is no longer supported. Use the cosmos.msg.v1.signer protobuf annotation instead.
- TxConfig has a new method SigningContext() *signing.Context.
- SigVerifiableTx.GetSigners() now returns ([][]byte, error) instead of []sdk.AccAddress.
- AccountKeeper now has an AddressCodec() address.Codec method and the expected AccountKeeper for x/auth/ante expects this method.
```

The only usage in cosmos is [here](https://github.com/Kava-Labs/cosmos-sdk/blob/f192238e804f4a36752a9ecf5087b090b3db52c4/baseapp/baseapp.go#L954-L960).

Reply:
yeah i looked into those briefly as well. It still not very clear how its used. will need some additional digging.